src/services/types.d.ts

```diff
@@ -0,0 +1,23 @@
+import { Config } from '../types';
+
+/**
+ * Interface for sending data to different platforms
+ */
+export interface InterfaceAPI {
```

Comment:
Let's use a more explicit name. It might be just `Service`. And please use this name in your file name. `Service.ts`